doc/introduction/interfaces/numpy.rst

```diff
@@ -154,7 +154,7 @@ and two non-trainable arguments ``data`` and ``wires``:
     dev = qml.device('default.qubit', wires=5)
 
     @qml.qnode(dev)
-    def circuit(weights, data, wires):
+    def circuit(weights, data, wires=None):
```

Comment:
```suggestion
    def circuit(weights, data, wires):
```

This can stay as is --- the workaround is just that it needs to be _passed_ as a kwarg!